src/appkit.cs

```diff
@@ -1174,13 +1201,16 @@ interface NSBezierPath : NSCoding, NSCopying {
 		[Export ("appendBezierPathWithArcFromPoint:toPoint:radius:")]
 		void AppendPathWithArc (CGPoint point1, CGPoint point2, nfloat radius);
 
+		[Obsolete ("Use 'AppendBezierPathWithCGGlyph' instead.")]
```

Comment:
Might be better to point to the new Append (glyphs, font) method instead of AppendBezierPathWithCGGlyphs?

Same for the next couple obsoletes